commit 6f1112d1404036a0bb88f77ab84c11c761507a42 (refs/changes/54/4854/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-09-19T15:46:30-07:00 (1 year, 1 month ago)
    
    TRITON-782 napi sdc-client test failure in nightly-2: "invalid MAC address"

diff --git a/test/napi.test.js b/test/napi.test.js
index 84e3c65..4d061a8 100644
--- a/test/napi.test.js
+++ b/test/napi.test.js
@@ -31,7 +31,7 @@ function pseudoRandomMac() {
         var diff = maxVal - minVal + 1.0,
             val = Math.random() * diff;
         val += minVal;
-        return Math.round(val);
+        return Math.floor(val);
     }
     mac[3] = randomInt(0x00, 0x7f);
     mac[4] = randomInt(0x00, 0xff);
